packages/core/database/src/metadata/relations.ts

```diff
@@ -120,6 +120,10 @@ const createOneToMany = (
   meta: Meta,
   metadata: Metadata
 ) => {
+  if (!shouldUseJoinTable(attribute)) {
```

Comment:
I would move that into the isBidirectional & throw if it's falsy as it's an error

Reply:
Actually this is the one that is necessary to suppress for our localizations hack. The localizations attribute tries to create a oneToMany, but we've set joinTable to false, so this allows it to skip it. I'll leave them both as returns but comment it so we remember why it was done.

Reply:
Would it make sense to add an option to disable database create instead in that case like `unstable__virtual: true`